website/docs/d/template.html.markdown

```diff
@@ -0,0 +1,49 @@
+---
+layout: "cloudstack"
+page_title: "Cloudstack: cloudstack_template"
+sidebar_current: "docs-cloudstack-datasource-template"
+description: |-
+  Get informations on a Cloudstack template.
+---
+
+# cloudstack_template
+
+Use this datasource to get the ID of a template for use in other resources.
+
+### Example Usage
+
+```hcl
+data "cloudstack_template" "my_template" {
+  template_filter = "featured"
+
+  filter {
+    name = "name"
+    value = "CentOS 7\\.1"
+  }
+
+  filter {
+    name = "hypervisor"
+    value = "KVM"
+  }
+}
+```
+
+### Argument Reference
+
+* `templatefilter` - (Required) The template filter (see the [Cloudstack documentation](https://cloudstack.apache.org/api/apidocs-4.9/apis/listTemplates.html))
```

Comment:
You missed the `_` in `templatefilter`

Reply:
And I would remove the link as the version keeps changing so pointing to a specific version means we need to maintain this link. I suggest to just give give the supported options here instead.